Doc: Created a separate topic for AWS authentication (#39012)
grafana/grafana: docs/sources/datasources/aws-cloudwatch/_index.md

Grafana ships with built-in support for CloudWatch. Add it as a data source, then you are ready to build dashboards or use Explore with CloudWatch metrics and CloudWatch Logs.

> This topic explains options, variables, querying, and other options specific to this data source. Refer to [Add a data source]({{< relref "../add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.

Reviewer: ```suggestion
This topic explains options, variables, and queries that are specific to this data source. For instructions about how to add a data source to Grafana, refer to [Add a data source]({{< relref "../add-a-data-source.md" >}}). Only users with the organization admin role can add data sources.
```
* What is the difference between "options" and "other options".
* Put the what before the reference for consistency.
* "on" means "about" here.

Reviewer: I am not sure about "querying" in the midst of those other nouns.

Author: Updated.

Grafana ships with built-in support for CloudWatch. Add it as a data source, then you are ready to build dashboards or use Explore with CloudWatch metrics and CloudWatch Logs.

> This topic explains queries, templates, variables and other configuration options specific to this data source. For instructions on how to add a data source to Grafan, refer to [Add a data source]({{< relref "../add-a-data-source.md" >}}). Only users with the organization admin role can add data sources.

Reviewer 2: ```suggestion
The following information describes queries, templates, variables, and other configuration specific to this data source. For instructions on how to add a data source to Grafana, refer to [Add a data source]({{< relref "../add-a-data-source.md" >}}). Only users with the organization admin role can add data sources.
```

Reviewer 2: The first sentence is pretty passive, but I don't think my suggestion really changes that. Toughy.

> > **Note:** If you have issues setting up the data source and Grafana is returning undescriptive errors, then check your log file (try looking in /var/log/grafana/grafana.log).

Reviewer 2: ```suggestion
> **Note:** If you have issues setting up the data source and Grafana is returning undescriptive errors, check your log file. This file can usually be found in `/var/log/grafana/grafana.log`.
```

## Authentication

> See [AWS authentication]({{< relref "aws-authentication.md" >}}) for available options and configuration details.

Reviewer: ```suggestion
For available options and configuration details, see [AWS authentication]({{< relref "aws-authentication.md" >}}).
```

Author: Done

### Dynamic queries using dimension wildcards

You can monitor a dynamic list of metrics by using the asterisk (\*) wildcard for one or more dimension values.

{{< figure src="/static/img/docs/v65/cloudwatch-dimension-wildcard.png" max-width="800px" class="docs-image--right" caption="CloudWatch dimension wildcard" >}}

> In this example, all metrics in the namespace `AWS/EC2` with a metric name of `CPUUtilization` and ANY value for the `InstanceId` dimension are queried. This can help you monitor metrics for AWS resources, like EC2 instances or containers. For example, when new instances get created as part of an auto scaling event, they will automatically appear in the graph without you having to track the new instance IDs. This capability is currently limited to retrieving up to 100 metrics. You can click on `Show Query Preview` to see the search expression that is automatically built to support wildcards. To learn more about search expressions, visit the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html).

Reviewer 3: ```suggestion
In this example, the query returns both all metrics in the namespace `AWS/EC2` with a metric name of `CPUUtilization` and ANY value for the `InstanceId` dimension. This type of query can help you monitor metrics for AWS resources, like EC2 instances or containers. For example, when new instances are created as part of an auto scaling event, the instances will automatically appear in the graph without needing to track the new instance IDs. This capability is currently limited to retrieving up to 100 metrics. Click `Show Query Preview` to see the search expression automatically built to support wildcards. To learn more about search expressions, visit the [CloudWatch documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/search-expression-syntax.html).
```

### Using template variables

As with several other data sources, the CloudWatch data source supports the use of template variables in queries.

> See the [Templating]({{< relref "../../variables/_index.md" >}}) documentation for an introduction to the templating feature and the different types of template variables.

Reviewer: ```suggestion
For an introduction to the templating feature and the different types of template variables, see the [Templating]({{< relref "../../variables/_index.md" >}}) documentation.
```

### Alerting

Since CloudWatch Logs queries can return numeric data, for example through the use of the `stats` command, alerts are supported.

> See the [Alerting]({{< relref "../../alerting/_index.md" >}}) documentation for more on Grafana alerts.

Reviewer: ```suggestion
For more about Grafana alerts, see the [Alerting]({{< relref "../../alerting/_index.md" >}}) documentation.
```

Author: Done.

## Templated queries

> Instead of hard-coding server, application and sensor names in your metric queries, you can use variables. The variables are listed as dropdown select boxes at the top of the dashboard. These dropdowns make it easy to change the display of data in your dashboard.

Reviewer 2: ```suggestion
Instead of hard-coding server, application, and sensor names in your metric queries, you can use variables. The variables are listed as dropdown select boxes at the top of the dashboard. These dropdowns make it easy to change the display of data in your dashboard.
```

Reviewer 2: Oxford comma NIT

Instead of hard-coding things like server, application and sensor name in your metric queries you can use variables in their place. Variables are shown as dropdown select boxes at the top of the dashboard. These dropdowns make it easy to change the data being displayed in your dashboard.

> See the [Templating]({{< relref "../../variables/_index.md" >}}) documentation for an introduction to the templating feature and the different types of template variables.

Reviewer: ```suggestion
For an introduction to the templating feature and the different types of template variables, see the [Templating]({{< relref "../../variables/_index.md" >}}) documentation.
```

Author: Done.

## Configure the data source with grafana.ini

> In the Grafana [configuration]({{< relref "../../administration/configuration.md#aws" >}}) file, there's an `AWS` section that allows you to customize the data source.

Reviewer 3: ```suggestion
The Grafana [configuration]({{< relref "../../administration/configuration.md#aws" >}}) file includes an `AWS` section where you can customize the data source.
```

## Configure the data source with provisioning

> You can configure the CloudWatch data sources using the configuration files with Grafana's provisioning system. To understand how it works and to learn more about available configuration options, refer to the [Provisioning Grafana]({{< relref "../../administration/provisioning/#datasources" >}}) topic.

Reviewer 3: I'm not sure what it refers to here, so my suggestion may not be correct:
```suggestion
You can configure the CloudWatch data sources using the configuration files with Grafana's provisioning system. To understand how the configuration file works and learn more about available configuration options, refer to the [Provisioning Grafana]({{< relref "../../administration/provisioning/#datasources" >}}) topic.
```

## Configure the data source with provisioning

> It's now possible to configure data sources using config files with Grafana's provisioning system. You can read more about how it works and all the settings you can set for data sources on the [provisioning docs page]({{< relref "../../administration/provisioning/#datasources" >}})

Reviewer: ```suggestion
It’s possible to configure data sources using configuration files with Grafana’s provisioning system. To understand how it works and to read about all of the settings for data sources, see [Provisioning data sources]({{< relref "../../administration/provisioning/#datasources" >}}).
```

Author: Updated.
